src/utils/forms-loader.test.ts

```diff
@@ -302,7 +302,7 @@ describe('Forms loader - getFormByVersion', () => {
   });
 });
 
-describe('Forms loader - applyFormIntent', () => {
+xdescribe('Forms loader - applyFormIntent', () => {
```

Comment:
what happened here?

Reply:
The change on the Form loader broke the test. Going to push updated review